src/pages/Register/Register.jsx

```diff
@@ -0,0 +1,41 @@
+import { useState } from "react";
+
+
+export default function Register() {
+    const [username, setUsername] = useState("");
+    const [email, setEmail] = useState("");
+    const [password, setPassword] = useState("");
+
+    const handleSubmit = (e) => {
+        e.preventDefault();
+        console.log("Registered");
+    };
+
+    return (
+        <form onSubmit={handleSubmit}>
+            <input
+                type="text"
+                value={username}
+                onChange={(e) => setUsername(e.target.value)}
+                required
+            />
+            <input
+                type="email"
+                value={email}
+                onChange={(e) => setEmail(e.target.value)}
+                placeholder="Email"
+                required
+            />
+            <input
```

Comment:
Add confirm password input and validate password and confirm password matches

Reply:
I added the confirm password, please confirm it